explanations/license-missing.md

```diff
@@ -0,0 +1,3 @@
+All packages should have a license.
+
+The `meta.license` attribute should preferrably contain a value from `lib.licenses` defined in [`nixpkgs/lib/licenses.nix`](https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix). See [Section 7.2 of Nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#sec-meta-license) for more details.
```

Comment:
Instead of section number, please use `Licenses section`. That should be more stable.